Gives every syndicate ghost role an evil survival box

AMyriad opened the pull request:
# Document the changes in your pull request
Adds a standard issue survival box to every ghost role working for the syndicate.

# Why is this good for the game?
Always bugged me that they never start with a survival box, and it makes sense that they should have one. And of course it has to be the evil variant.

# Testing
Not tested yet, I'll do it soon when I can.

# Wiki Documentation
I don't think this kinda stuff is documented anywhere.

# Changelog
:cl:  
tweak: Syndicate ghost roles now have a survival box
/:cl:

AMyriad committed:
woe, box upon ye

AMyriad committed:
no one is safe, box upon ye all

Majkl-J reviewed (commented):
Otherwise looks fine, bonus points for formatting

AMyriad committed:
apparently none of them had bags lol?

also no order to what's in the outfit so I just but the backpacks at the bottom

AMyriad committed:
fixes ordering yes

also what apparently they don't need bags on their back for it to be filled with stuff
